packages/unit-testing/stubs/crypto/crypto.js

```diff
@@ -0,0 +1,145 @@
+     * @since 2015.1
+     */
+    crypto.prototype.createSecretKey = function (options) { };
+
+    /**
+     * Method used to create a crypto.Hash object.
+     * @param {Object} options
+     * @param {string} options.algorithm The hash algorithm. Set using the crypto.HashAlg enum.
+     * @return {Hash}
+     * @throws {SuiteScriptError} MISSING_REQD_ARGUMENT If options or mandatory parameter are undefined.
+     * @since 2015.1
+     */
+    crypto.prototype.createHash = function (options) { };
+
+    /**
+     * Method used to create a crypto.Hmac object.
+     * @param {Object} options The options object.
+     * @param {string} options.algorithm The hash algorithm. Use the crypto.HashAlg enum to set this value.
+     * @param {SecretKey} options.key The crypto.SecretKey object.
+     * @return {Hmac}
+     * @throws {SuiteScriptError} MISSING_REQD_ARGUMENT If options or mandatory parameter are undefined.
+     * @since 2015.1
+     */
+    crypto.prototype.createHmac = function (options) { };
+
+    /**
+     * Method used to create and return a crypto.EncryptionAlg object.
+     * @governance none
+     * @param {object} options The options object.
+     * @param {string} options.algorithm The hash algorithm. Set the value using the crypto.EncryptionAlg enum.
+     * @param {SecretKey} options.key The crypto.SecretKey object.
+     * @param {string} options.blockCipherMode
+     * @param {string} [options.padding] The padding for the cipher text.
+     * @return {Cipher}
+     * @throws {SuiteScriptError} MISSING_REQD_ARGUMENT If options or mandatory parameter are undefined.
+     * @since 2015.1
+     */
+    crypto.prototype.createCipher = function (options) { };
+
+    /**
+     * Method used to create a crypto.Decipher object.
+     * @param {object} options The options object.
+     * @param {string} options.algorithm The hash algorithm. Set by the crypto.EncryptionAlg enum.
+     * @param {string} options.key The crypto.SecretKey object used for encryption.
+     * @param {string} options.blockCipherMode
+     * @param {Object} [options.padding] The padding for the cipher. Set the value using the crypto.Padding enum.
+     * @param {string} options.iv The initialization vector that was used for encryption.
+     * @return {Decipher}
+     * @throws {SuiteScriptError} MISSING_REQD_ARGUMENT If options or mandatory parameter are undefined.
+     * @since 2015.1
+     */
+    crypto.prototype.createDecipher = function (options) { };
+
+    /**
+     * Method to check whether a password in a record corresponds to the input value
+     * @param {object} options The options object.
+     * @param {string} options.recordType Type of the record with a password field
+     * @param {number} options.recordId Id of the record with password field
+     * @param {value} options.value Input value to be checked against the password stored in the record
```

Comment:
value should be string
https://nlcorp.app.netsuite.com/app/help/helpcenter.nl?fid=section_160806904480.html

Reply:
Done